### ISSUE TITLE
Setting up NFT as governance

### ISSUE THREAD

**Issue by july5613** [FantomMenaceDAO/MIPs#3]
# Menace Improvement Proposal (MIP-3)

## tl;dr
Give ourselves NFTs as governance tokens to vote

## Scope
What is the scope of the project? What will it require?
- This is TBD

## Motivation
We should this because...
- We need to vote on things with an NFT. 
- We need to automate the onboarding process further so we can onboard more folks 
- so we can vote w/o having to verify whether they belong to the DAO or not

## Owners
Who will be owners for the project and be responsible for delivering the project
- @0xJuly (dev, gov) 
- @NaturalVoids (gov, art) 
- @hashyjim (art)
- so far
- open to other folks joining (put your username above)

## Resources / Compensation
Do you need funding? Do the contributors of the project need to be paid in NFTs or FTM? 
- We will need to compensation is TBD

## Timeline
How long will it this roughly take?
- TBD

### Additional context
- More to come

**Comment by armenarmen** [comment]
yes.
